sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/BulkPatchItemRequestOptions.java

```diff
@@ -0,0 +1,88 @@
+package com.azure.cosmos;
```

Comment:
header file is missing. please add the header file to this new file and all other new files.